lib/computer.rb

```diff
@@ -0,0 +1,21 @@
+class Computer
+  attr_reader :symbol
+
+  def initialize(symbol:, opponent:, pauser:)
+    @symbol = symbol
+    @opponent = opponent
+    @pauser = pauser
+  end
+
+  def play(game)
+    pauser.pause
```

Comment:
Let's keep an eye on this. I am wondering pauser transitions into something more generic. No action need for now though.